ui/templates/main.html

```diff
@@ -1,94 +1,100 @@
-<!DOCTYPE html><html><head><title>Φορτίο v{{.Version}}</title>
-<script src="{{.ChartJSPath}}"></script>
-<script src="{{.Version}}/static/js/fortio_chart.js"></script>
-<link rel="icon" href="../favicon.ico" />
+<!DOCTYPE html>
+<html>
+<head><title>Φορτίο v{{.Version}}</title>
+  <script src="{{.ChartJSPath}}"></script>
+  <script src="{{.Version}}/static/js/fortio_chart.js"></script>
+  <link rel="icon" href="../favicon.ico"/>
 </head>
 <body style="background: linear-gradient(to right, #d8aa20 , #c75228);">
-<a href="https://fortio.org/" target="_blank"><img src="{{.LogoPath}}" alt="." height="69" width="45" align="right" /></a>
-<img src="{{.LogoPath}}" alt="." height="92" width="60" align="right" />
-<img src="{{.LogoPath}}" alt="istio logo" height="123" width="80" align="right" />
+<a href="https://fortio.org/" target="_blank"><img src="{{.LogoPath}}" alt="." height="69" width="45"
+                                                   align="right"/></a>
+<img src="{{.LogoPath}}" alt="." height="92" width="60" align="right"/>
+<img src="{{.LogoPath}}" alt="istio logo" height="123" width="80" align="right"/>
 <h1>Φορτίο (fortio) v{{.Version}}{{if not .DoLoad}} control UI{{end}}</h1>
 <p>Up for {{.UpTime}} (since {{.StartTime}}).
 {{if .DoLoad}}
 <p>{{.Labels}} {{.TargetURL}}
 <div id="running">
-Running load test... Results pending...
-<br />
```

Comment:
please leave as xhtml `<br />` (or we could switch to plain html `<br>` but not `<br/>`)
https://www.quora.com/What-is-the-difference-between-br-and-br-tag-in-HTML-Which-one-should-we-use

also please run through weblint (from the running fortio server) before/after

Reply:
Done